### ISSUE TITLE
Error installing

### ISSUE THREAD

**Issue by tildebyte** [Calysto/skulpt_python#1]
Running `sudo pip install --upgrade skulpt_python`

yields

``` py
  File "/tmp/pip_build_root/skulpt-python/setup.py", line 49, in <module>

    with open('README.md') as f:

IOError: [Errno 2] No such file or directory: 'README.md'

```

**Comment by dsblank** [comment]
Oops. Thanks.

**Comment by dsblank** [comment]
Fixed with latest upload to pypi, version 0.9.2.
